src/dataflow_pipeline.py

```diff
@@ -239,31 +246,40 @@ def cdc_apply_changes(self):
             struct_schema = modified_schema
 
         if cdc_apply_changes.scd_type == "2":
-            for field in struct_schema.fields:
-                if field.name == cdc_apply_changes.sequence_by:
-                    struct_schema.add(StructField("__START_AT", field.dataType))
-                    struct_schema.add(StructField("__END_AT", field.dataType))
-                    break
+            struct_schema.add(StructField("__START_AT", IntegerType()))
+            struct_schema.add(StructField("__END_AT", IntegerType()))
 
         dlt.create_streaming_live_table(
             name=f"{self.dataflowSpec.targetDetails['table']}",
+            table_properties=self.dataflowSpec.tableProperties,
             partition_cols=DataflowSpecUtils.get_partition_cols(self.dataflowSpec.partitionColumns),
             path=self.dataflowSpec.targetDetails["path"],
             schema=struct_schema,
         )
 
+        apply_as_deletes = None
+        if cdc_apply_changes.apply_as_deletes:
+            apply_as_deletes = expr(cdc_apply_changes.apply_as_deletes)
```

Comment:
what's the rationale of creating apply_as_deletes/truncates as separate variables? given that the other parameters still seem to use cdc_apply_changes?

Reply:
`apply_changes(
  target = "<target-table>",
  source = "<data-source>",
  keys = ["key1", "key2", "keyN"],
  sequence_by = "<sequence-column>",
  ignore_null_updates = False,
  apply_as_deletes = None,
  apply_as_truncates = None,
  column_list = None,
  except_column_list = None,
  stored_as_scd_type = <type>,
  track_history_column_list = None,
  track_history_except_column_list = None
)`

This is definition of public dlt appy_changes api which we mapped.